lib/libvfio-user.c

```diff
@@ -555,6 +562,7 @@ handle_device_get_region_io_fds(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg)
     ioeventfd_t *sub_reg = NULL;
     size_t nr_sub_reg = 0;
     size_t i = 0;
+    size_t nr_shdw_reg = 0;
```

Comment:
nit, s/shdw/shadow/ ? The shorter version doesn't seem better here

Reply:
ok